pkg/client/imagestreams.go

```diff
@@ -71,7 +71,7 @@ func (c *imageStreams) Get(name string) (result *imageapi.ImageStream, err error
 // GetByNamespace returns information about a particular image stream in a particular namespace and error if one occurs.
 func (c *imageStreams) GetByNamespace(namespace, name string) (result *imageapi.ImageStream, err error) {
 	result = &imageapi.ImageStream{}
-	c.r.Get().Namespace(namespace).Resource("imageStreams").Name(name).Do().Into(result)
+	err = c.r.Get().Namespace(namespace).Resource("imageStreams").Name(name).Do().Into(result)
```

Comment:
GetByNamespace doesn't seem to be used anywhere so I would suggest removing it altogether.